var/spack/repos/builtin/packages/quantum-espresso/package.py

```diff
@@ -88,6 +96,27 @@ class QuantumEspresso(Package):
     # folder QE expects as a link, we issue a conflict here.
     conflicts('+elpa', when='@:5.4.0')
 
+    # The first version of Q-E to feature integrated EPW is 6.0.0,
+    # as per http://epw.org.uk/Main/DownloadAndInstall .
+    # Complain if trying to install a version older than this.
+    conflicts('+epw', when='@:5',
+            msg='EPW only available from version 6.0.0 and on')
+
+    # Below goes some constraints as shown in the link above.
+    # Constraints may be relaxed as successful reports
+    # of different compiler+mpi combinations arrive
+
+    # KISS for now and only build the mpi version
```

Comment:
I think it would be more useful to phrase this as a TODO like `TODO: add support for X in the future also`

Reply:
Fixed by 299e4d9